examples/models/llama2/custom_ops/CMakeLists.txt

```diff
@@ -25,7 +25,7 @@ if(NOT TORCH_ROOT)
   set(TORCH_ROOT ${EXECUTORCH_ROOT}/third-party/pytorch)
 endif()
 
-set(_common_compile_options -Wno-deprecated-declarations)
+set(_common_compile_options -Wno-deprecated-declarations -fPIC)
```

Comment:
Please add a comment explaining why this sets `-fPIC` manually instead of using `CMAKE_POSITION_INDEPENDENT_CODE`

Reply:
I try to avoid using `CMAKE_POSITION_INDEPENDENT_CODE` as much as possible since it's a global variable. We have to use it for third-party libs because we can't change their CMakeLists.txt but for this one we can tweak the compile options for this specific lib.